protocols/migration/migration_helpers.py

```diff
@@ -62,41 +65,55 @@
 class MigrationHelpers(object):
 
     @staticmethod
-    def migrate_interpretation_request_rd_to_latest(json_dict, assembly):
+    def migrate_interpretation_request_rd_to_latest(json_dict, assembly=None):
```

Comment:
I realise this might not be the moment, but could this not be done as recursion or something. There's a lot of duplication in this class generally.

Reply:
I was thinking something like: 

def migrate(json_dict, models, migrations):
    if PayloadValidation(klass=models[0], payload=json_dict).is_valid:
        return model[0].fromJsonDict(jsonDict=json_dict)
    else:
       return migrations[0].migrate_interpretation_request_rd(migrate(json_dict, models[1:], migrations[1:]))

Might not be exactly right.